src/js/pagination/paging-collection.js

```diff
@@ -343,11 +345,9 @@
                     value = value.join(',');
                 }
                 this.filterableFields[fieldName].value = value;
-                if (_.isUndefined(this.queryParams.fieldName)) {
```

Comment:
It wasn't ever needed; I mistakenly added it in order to not have to redefine the same function over and over. However I ran into some edge cases in my consumer code where changing the value of a filter didn't actually result in the new (rather the old) value being sent in the subsequent request. Removing this conditional fixed that issue. Without diving too deep into the code or thinking about it too hard I'm pretty sure it had to do with JS's odd closure implementation.